src/components/QuranReader/PageView/groupLinesByVerses.ts

```diff
@@ -22,7 +22,12 @@ const groupLinesByVerses = (verses: VerseType[]) => {
   });
 
   // Groups the words based on their (page and) line number
+  /* TODO: (@naveed-ahmad) use v1Page, v2page instead. Depneds on font type use has selected.
+       OR change the backend to send pageNumber based on font type user has requested
+  */
   const lines = _.groupBy(words, (word) => {
+    // eslint-disable-next-line no-param-reassign
```

Comment:
Actually going to change the api, if client requests 'v2' code, api will return `pageNumber` of v2 Mushaf. Removing `v1Page` and `v2Page`. This will simplify the frontend a lot.

Reply:
perfect.